src/actions/requests.rs

```diff
@@ -904,6 +908,23 @@ impl RequestAction for CodeLensRequest {
     }
 }
 
+fn racer_cache(vfs: Arc<Vfs>) -> racer::FileCache {
+    struct RacerVfs(Arc<Vfs>);
```

Comment:
Why you use wrapper struct here?

Reply:
I can’t implement he trait directly on Vfs due to coherence.

This was previously in rls_vfs without wrapper, but 
https://github.com/rust-lang-nursery/rls/pull/964 changes that, so we can avoid bumping racer version in rls_vfs

Reply:
I see. Thanks.